web/app/themes/mitlib-child/inc/content-widgetized.php

```diff
@@ -13,17 +15,20 @@
 	
 	<div class="entry-content">
 		<?php
-		$title = get_the_title();
-		if ( '' !== $title ) :
+		$page_title = get_the_title();
+		if ( '' !== $page_title ) :
 			if ( ! is_front_page() ) {
-				echo '<h1>' . esc_html( $title ) . '</h1>'; }
+				echo '<h1>' . esc_html( $page_title ) . '</h1>'; }
 		endif;
 
 		the_content();
 
+		/**
+		 * The "sidebar-two" region is named "Below Content Widget Area" within
+		 *  the admin interface.
+		 */
 		dynamic_sidebar( 'sidebar-two' );
```

Comment:
Is the only difference between this and `inc/content-page.php` that the `dynamic_sidebar` is not conditional here and tries to display even if not active?

Reply:
That conditional doesn't actually seem needed - `dynamic_sidebar()` returns nothing if there are no active widgets in that sidebar, so the returned markup is identical in that regard.

The only difference between these two partials at the moment is that `inc/content-page.php` adds a `content-main` class to one of the divs, so you get `<div class="main-content content-main">` instead of `<div class="main-content">`. This does seem relevant to some CSS rules around the site, but I'm not sure if this is something that UX will want to preserve or not.

Reply:
Interesting. One CSS class difference definitely feels like a "this template is gonna get booted" kinda a think. Later is fine though :)